### DISCUSSION TITLE
Remove achievements from the website. They make no sense and only distract users who are trying to do work.

### DISCUSSION THREAD

**Discussion by simon-glitch** [community/community#113028] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It's all in the title.

**Comment by SebastienXia** [comment]
Hello, @simon-glitch 

> If you don't like achievements, you can hide them of your profile. Via this link => https://github.com/SebastienXia?tab=achievements

**Reply by electrovir** [reply]
This does not hide achievements on others' profiles nor does it remove the incentive for the entire rest of the users on GitHub to abuse to system in order to earn achievements.

**Reply by ethanwater** [reply]
Just don't look at them.

**Comment by jfullstackdev** [comment]
we have different preferences, right ? so someone who don't want it can simply ignore it, GitHub offers so many other features that of course if others are not using, can simply be ignored
